libraries/wallet/wallet.cpp

```diff
@@ -838,10 +838,13 @@ class wallet_api_impl
       }
       catch(...)
       {
-         string ws_password = _wallet.ws_password;
-         _wallet.ws_password = "";
-         wlog("wallet file ${data}", ("data", fc::json::to_pretty_string( _wallet ) ) );
-         _wallet.ws_password = ws_password;
+         if (&_wallet != nullptr) 
```

Comment:
&_wallet can never be null. It's a struct member of *this.

Reply:
it fixed travis error with violation memory access in cli_test as was before, strange behavior of travis that's why I've added this check

Reply:
The travis error is unrelated, please undo this change.

Reply:
ok sure